post-processor/vagrant/libvirt_test.go

```diff
@@ -70,3 +74,81 @@ func Test_sizeInMegabytes_WithExabytesUnit(t *testing.T) {
 	assertSizeInMegabytes(t, "1234E", 1234*1024*1024*1024*1024)
 	assertSizeInMegabytes(t, "1E", 1*1024*1024*1024*1024)
 }
+
+func Test_ManyFilesInArtifact(t *testing.T) {
+	p := new(LibVirtProvider)
+	ui := testUi()
+	type testCases struct {
+		Files         []string
+		Format        string
+		FilesExpected []string
+	}
+	testcases := []testCases{
+		{
+			[]string{},
+			"qcow2",
+			[]string{},
+		},
+		{
+			[]string{"test"},
+			"vmdk",
+			[]string{"box_0.img"},
+		},
+		{
+			[]string{"test", "/test-1", "test-2"},
+			"qcow2",
+			[]string{"box_0.img", "box_1.img", "box_2.img"},
+		},
+		{
+			[]string{"test", "efivars.fd", "/test-1", "test-2"},
```

Comment:
I wonder, is this representative of what is forwarded? From what I see the disks are created with the output directory, followed by the name of the disk, and subsequent disks are `<disk-name>-<disk-id-1-based>`.
There should never be a disk with an absolute path (unless someone invokes the builder with `/` as the output directory, which would be very questionable), and if there is, all the disks will point to the same directory.
That being said, I could be wrong, and if you experienced such a behaviour, could you share a template that produces such a result? This could be a bug in the qemu plugin fwiw

Reply:
I remove `/` for in my test.
The `/` was for represented directory without complexify the test. But it's not necessary.